tsbrowse/model.py

```diff
@@ -54,44 +43,35 @@ def __init__(self, tsbrowse_path):
             "migrations",
             "provenances",
         ]:
+            ts_table = getattr(self.ts.tables, table_name)
```

Comment:
That's generating a fresh copy of the tables each time. Get a reference to the tables before the loop body

Reply:
Fixed!